### DISCUSSION TITLE
【質問】「TOPPERS BASE PLATFORM対応版基礎1、2、3セミナー教材のダウンロード」について

### DISCUSSION THREAD

**Discussion by hobata** [toppers/users-forum#139] [TOPPERS Seminar]
始めまして。

勝手ながら、表題の資料を勤務先の研修で参考にしたいと思っています。
ちなみに、Raspberry Pi picoに対応した、
「TOPPERS BASE PLATFORM対応版基礎1、2、3セミナー教材のダウンロード」は可能でしょうか。
もし公開済みでしたら、お手数ですがURLを教えてください。
また未公開の場合は、お手数ですが、ご検討を、よろしくお願いします。

**Comment by Ryohsuke-Takeuchi** [comment]
Raspberry Pi picoに対応した教材は、新基礎３、上級１、２を予定しています。
新基礎３は、Nucleo版の基礎３の内容をpico用に修正したものです。
pico以外にTEB003ボードが必要です。（インターフェイス誌で４月頃配布されると思います）
また、シールドはTEB002ではなく、LCDJOYシールドに変更しています。
現在のところ、基礎１，２のセミナーに対応する予定はありません。
新基礎３、上級１、２の公開予定は、今年の４月頃です。
よろしくお願いいたします。
